File: cfgs/java_cfgs_foo/COUNT_OPERATIONS_MAKE_STRINGAB_FREE.java
Nodes
1;special;;;
2;statement;int b_count = 0;14;
3;statement;int res = 0;15;
4;statement;int i = 0;16;
5;conditional;foo_1 < foo_2.length;17;
6;conditional;foo_1[(foo_2.length - foo_3) - 1] == 'a';19;
7;statement;res = foo_1 + foo_2;20;
8;statement;b_count = foo_1 * 2;21;
9;statement;b_count += 1;24;
10;statement;i++;18;
11;statement;return foo_1;27;
12;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;11;
5;7;9;
6;8;
7;10;
8;10;
9;5;
10;12;
11;

File: cfgs/pythoncfgs/COUNT_OPERATIONS_MAKE_STRINGAB_FREE.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = 0;9;
3;conditional;fun_2(fun_2(foo_1));10;
4;statement;return foo_1;16;
5;conditional;foo_1[~foo_2] == 'a';11;
6;statement;foo_1 += 1;15;
7;statement;foo_1 = foo_2 + foo_3;12;
8;statement;foo_1 = foo_2 * 2;13;
9;exit;;
Edges
0;2;
1;3;
2;4;5;
3;9;
4;6;7;
5;3;
6;8;
7;3;
8;
